### PR TITLE
Added "which" command documentation

### LINKED ISSUE #63: which

### PR DESCRIPTION
**Pull request by leonliang0395**
# Advanced HW 9 submission

Closes #63  

Name: Leon Liang
Uniqname: lileon

### CODE REVIEW

**Comment by the-alex** [comment]
Be sure to include the line `<!---more--->`. Everything above that line will show as a preview, similar to the other commands on [the c4cs website](https://c4cs.github.io/reference). Right now, your entire entry is displayed on the reference page.

for example, here is the entry for `clear` ...

~~~markdown
---
---

clear
-------

`clear` clears the terminal screen

\~~~ bash
$ clear
$
\~~~

<!--more-->

### Useful Options / Examples

* You can use CTRL+L to acheive the same effect
~~~

File: _commands/basics/which.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+~~~ bash
+$  which ls
+/bin/ls
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+### `which cd ls vim emacs which`
```
Comment (mterwill) [inline _commands/basics/which.md]: ``cd`` isn't below

**Comment by mterwill** [comment]
Talk about builtins and why they don't show up.

**Comment by mterwill** [comment]
One minor nitpick: Lines 76 + 90 are h3 as opposed to the h5 you'd  been using for 'Break it down' above. Other than that, :+1:

**Comment by mterwill** [comment]
Thanks! Looks great.

**Pull request merged**